src/test_demo/updatedDetectionGreyScale.py

```diff
@@ -59,10 +65,19 @@
 
         frame = cv2.polylines(frame, [np.int32(dst)], True, (0, 255, 0), 3, cv2.LINE_AA)
         print("Match found!")
+        break
+    else:
+        print("Not matching")
 
     # Display the video feed with detected template
     cv2.imshow('Detected Template', frame_matches)
 
+
+    # 카메라가 10초가 지나면 종료
+    elapsed_time = time.time() - start_time
+    if elapsed_time > 10:
```

Comment:
How about changing elasped time to 20sec not 10?
I think program can't figure out fake one until 10secs.

Reply:
Okay, I will change it right now